internal/integration/mysql_test.go

```diff
@@ -1278,7 +1278,7 @@ func (t *myTest) dsn(dbname string) string {
 	if t.mariadb() {
 		d = "mariadb"
 	}
-	return fmt.Sprintf("%s://root%s@localhost:%d/%s", d, pass, t.port, dbname)
```

Comment:
Remove and let Atlas handle it for the users.

Reply:
This is in the test without a client open in between. Here it is valid to add the parseTime by hand.